### DISCUSSION TITLE
API Pending status.

### DISCUSSION THREAD

**Discussion by Kaustav2022** [deepgram/community#478] [General help]
My angular code is: -

  transcribeFile() {
    console.log("come");
    
    const options = {
      smart_format: false,
      model: "nova-2"
    };

    const file = "../../../assets/WhatsApp Ptt 2023-12-04 at 11.12.24 PM.wav";
    const deepgramApiKey = this.apiKey;
    const url = this.apiUrl;

    const formData = new FormData();
    formData.append('file', new File([file], 'audio.wav', { type: 'audio/wav' }));
    formData.append('options', JSON.stringify(options));

    const headers = new HttpHeaders()
      .set('Authorization', 'Token ' + deepgramApiKey)
      .set('Content-Type', 'undefined');

    this.http.post(url, formData, { headers: headers })
      .subscribe(
        (response) => {
          console.dir(response, { depth: null });
        },
        (error) => {
          console.error(error);
        }
      );
  }
  
  when call this function the API response are pending.
  
 
![image](https://github.com/deepgram/community/assets/98321736/7a3bb9a3-7621-4eba-acc8-fe3cab8d8fde)

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
